### PR TITLE
[MASTG-TOOL-0103] add uber-apk-signer

### LINKED ISSUE #2438: [new tool] uber-apk-signer
An easy tool to sign APKs:

https://github.com/patrickfav/uber-apk-signer

### PR DESCRIPTION
**Pull request by cpholguera**
Closes #2438

### CODE REVIEW

**Comment by cpholguera** [comment]
hey @ScreaMy7, I just noticed that you had it assigned. Would you mind reviewing? I can assign you another one. Sorry about that!

File: tools/android/MASTG-TOOL-0103.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: uber-apk-signer
+platform: android
+source: https://github.com/patrickfav/uber-apk-signer
+---
+
+A tool that helps to sign, [zip aligning](https://developer.android.com/studio/command-line/zipalign.html) and verifying multiple Android application packages (APKs) with either debug or provided release certificates (or multiple). It supports v1, v2, [v3](https://source.android.com/security/apksigning/v3) and [v4](https://source.android.com/security/apksigning/v4) Android signing schemes. Easy and convenient debug signing with embedded debug keystore. Automatically verifies signature and zipalign after every signing.
```
Comment (TheDauntless) [inline tools/android/MASTG-TOOL-0103.md:7]:
```suggestion
A tool that helps to sign, [zip align](https://developer.android.com/studio/command-line/zipalign.html) and verify one or more Android application packages (APKs) with either debug or provided release certificates. It supports v1, v2, [v3](https://source.android.com/security/apksigning/v3) and [v4](https://source.android.com/security/apksigning/v4) Android signing schemes. Easy and convenient debug signing with embedded debug keystore. Automatically verifies signature and zip alignment after signing.
```

File: tools/android/MASTG-TOOL-0103.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Download the jar from the [latest release](https://github.com/patrickfav/uber-apk-signer/releases/latest) and run:
+
+```bash
+java -jar uber-apk-signer.jar --apks /path/to/apks
```
Comment (TheDauntless) [inline tools/android/MASTG-TOOL-0103.md:12]:
```suggestion
$ java -jar uber-apk-signer.jar --apks </path/to/apks>
```
(I noticed this in https://mas.owasp.org/contributing/5_Style_Guide/#8-code-and-shell-commands )

File: tools/android/MASTG-TOOL-0103.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+```
+
+Demo:
+
+[![asciicast](https://asciinema.org/a/91092.png)](https://asciinema.org/a/91092)
```
Comment (TheDauntless) [inline tools/android/MASTG-TOOL-0103.md:17]: Do we want to go this route, vs including an example ourselves?

Comment (cpholguera) [reply tools/android/MASTG-TOOL-0103.md:17]: This is fine, whenever there are examples available we can include them. If there are videos, even better!

**Review by TheDauntless — APPROVED** [review]
LGTM!

**Pull request merged**